### PR TITLE
Add optional decimal places argument

### PR DESCRIPTION
**Pull request by filiptronicek**
Add an argument, which will get rid of some decimal places for the number to be as precise, as you want it to.
```typescript
randomFloat(10, 1000, 5) //  88.22269
```

### CODE REVIEW

**Comment by yg** [comment]
Thanks for the PR @filiptronicek!

:purple_heart:

**Comment by yg** [comment]
@filiptronicek Can you update README too?

**Comment by filiptronicek** [comment]
I hope it's a useful addition, will update it in a minute

File: README.md [hunk 1]
```diff
@@ -17,17 +17,24 @@ randomFloat(10);
 
 randomFloat(10, 1000);
 //=> 79.99198247622843
+
+randomFloat(10, 1000);
```
Comment (yg) [inline README.md:21]:
```suggestion
randomFloat(10, 1000, 5);
```

File: README.md [hunk 2]
```diff
@@ -43,6 +50,11 @@ Default: `1`
 
 Maximum float to return.
 
+#### places
+
+Type `number`
```
Comment (yg) [inline README.md:55]: Two spaces for a new line

```suggestion
Type `number`  
```

File: README.md [hunk 3]
```diff
@@ -43,6 +50,11 @@ Default: `1`
...
+Default: 15
```
Comment (yg) [inline README.md:56]:
```suggestion
Default: `15`
```

**Comment by yg** [comment]
Everything looks good! :tada:

**Pull request merged**